### PR TITLE
Chore: Add 'how to install' to packages

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messagess)

For non-trivial changes, please make sure you also:

- [x] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

File: packages/configuration-web-recommended/README.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ production websites.
 
 This package will automatically install all the missing dependencies.
 
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @sonarwhal/configuration-web-recommended
+```
+
+If you want to install it globally, add the parameter `-g`.
```
Comment (alrra) [inline packages/configuration-web-recommended/README.md]: Maybe we should just link to the [npm docs](https://docs.npmjs.com/cli/install) docs?

Comment (alrra) [reply packages/configuration-web-recommended/README.md]: Or have something in the lines of:

> You can make `npm` install it as a `devDependency` using the `--save-dev` parameter, or to install it globally, you can use the `-g` parameter. For other options see the [`npm`'s documentation](https://docs.npmjs.com/cli/install).

Comment (sarvaje) [reply packages/configuration-web-recommended/README.md]: Do you mean something like:
`For more install options, you can see the [npm documentation](https://docs.npmjs.com/cli/install)`
?

**Comment by sarvaje** [comment]
@alrra done!

File: packages/configuration-web-recommended/README.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ production websites.
...
 
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @sonarwhal/configuration-web-recommended
+```
+
+You can make `npm` install it as a `devDependency` using the `--save-dev`
+parameter, or to install it globally, you can use the `-g` parameter. For
+other options see the
```
Comment (alrra) [inline packages/configuration-web-recommended/README.md]: Drop `the`.

File: packages/configuration-web-recommended/README.md [hunk 3]
```diff
@@ -5,6 +5,17 @@ production websites.
 
 This package will automatically install all the missing dependencies.
 
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @sonarwhal/configuration-web-recommended
+```
+
+You can make `npm` install it as a `devDependency` using the `--save-dev`
```
Comment (alrra) [inline packages/configuration-web-recommended/README.md]: Note: You can...

**Comment by sarvaje** [comment]
@alrra done!

**Pull request closed without merging**